### ISSUE TITLE
Siri works but home app doesn’t

### ISSUE THREAD

**Issue by ldhall1856** [milo526/homebridge-tuya-web#65]
So I've been experiencing a lot of issues the past few days.  I know there are kinks to be worked out.  

I went to the home app but the device I wanted to control were "not responding" But I decided I would try Siri to turn on the device and it worked... 

Not sure what this means.... I'm not a coder by any means just thought it might help other issues.

**Comment by cilianospina** [comment]
Try 
`sudo npm install -g @milo526/homebridge-tuya-web@0.4.7-0`

**Comment by tasict** [comment]
I have the same issue and devices are non-responsive. Log shows [TuyaWebPlatform] [GET] Requesting too quickly. - you can query once in 60 seconds.

**Comment by milo526** [comment]
Are you sure you both are running the latest pre-release as @cilianospina proposed?
This will not (yet) be listed as a normal update, just running `npm update` will not work!
Neither will it show up in config-ui-x.

**Comment by ldhall1856** [comment]
I'm 90% sure I'm on the pre-release.  I ran that package at 8pm before I submitted a ticket.￼￼

**Comment by milo526** [comment]
Sorry but I can't help you base on a 90% assurance, debugging these kinds of issues is very time consuming for me.
Please try to figure out what the issue might be for yourself first.

In the log files there will be the following few lines:

```
[19-8-2020 19:02:27] Loaded plugin: @milo526/homebridge-tuya-web@0.4.7-0
[19-8-2020 19:02:27] Registering platform '@milo526/homebridge-tuya-web.TuyaWebPlatform'
```

Please assure that the loaded plugin is `...@0.4.7-0` and not any older version (probably `0.4.6`)

**Comment by tasict** [comment]
> I have the same issue and devices are non-responsive. Log shows [TuyaWebPlatform] [GET] Requesting too quickly. - you can query once in 60 seconds.

My problem has been solved after sudo npm install -g @milo526/homebridge-tuya-web@0.4.7-0

**Comment by milo526** [comment]
This issue is solved in [v0.4.7](http://github.com/milo526/homebridge-tuya-web/releases/tag/v0.4.7)

If you have any other issues or are still experiencing the issue documented here, please open a new issue.

If you use this plugin heavily please consider [becoming a sponsor](https://github.com/sponsors/milo526) or [making a one time donation](https://bunq.me/HomebridgeTuyaWeb)
